src/Jmikola/AutoLogin/Http/Firewall/AutoLoginListener.php

```diff
@@ -62,10 +67,21 @@ public function __construct($securityContext, AuthenticationManagerInterface $au
     }
 
     /**
+     * Sets the RememberMeServices implementation to use.
+     *
+     * @param RememberMeServicesInterface $rememberMeServices
+     */
+    public function setRememberMeServices(RememberMeServicesInterface $rememberMeServices)
+    {
+        $this->rememberMeServices = $rememberMeServices;
+    }
+
+    /**
      * @see Symfony\Component\Security\Http\Firewall\ListenerInterface::handle()
      */
     public function handle(GetResponseEvent $event)
     {
+        $requestEvent = $event;
```

Comment:
Assuming this was necessary because I re-assign `$event` in the code below, let's just change the argument name to `$requestEvent` and modify the line below to `$request = $requestEvent->getRequest();`.